app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
viết ngắn gọn nếu action ko có xử lý bên trong nhé
```suggestion
  def home; end
```